sdk/core/azure-core/src/http/log_policy.cpp

```diff
@@ -89,7 +89,7 @@ Azure::Core::CaseInsensitiveSet const
         "traceparent",
         "tracestate",
         "Transfer-Encoding",
-        "User-Agent"
+        "User-Agent",
```

Comment:
If we ship `beta.2` out of band (which we may if we really want all these changes for Attestation this month - let's discuss offline), then this fix needs to be mentioned in the changelog.